raiden/blockchain_events_handler.py

```diff
@@ -326,17 +326,20 @@ def handle_channel_closed(raiden: "RaidenService", event: Event):
             raiden.handle_and_track_state_change(channel_closed)
         else:
             # Must be a light client
```

Comment:
could you explain why did you put this docstring "Must be a light client"? 

This handler function is both for managing a closed channel for a full node and a light client

Reply:
if the nodes address (raiden.address) is not the same as the chanel_state.our_state.address, then, necessarilly, the node is a light client